pkg/git/poll/handler_test.go

```diff
@@ -164,6 +167,28 @@ var _ = Describe("Gitrepo pooling tests", func() {
 				fetcher.EXPECT().LatestCommit(gomock.Any(), gomock.Any(), gomock.Any()).Return("commit", nil).Times(1)
 			},
 		),
+
+		Entry("gitrepo present, same polling interval, disablePolling=false, generation changed",
+			10*time.Second, false, true, func(gitRepo v1alpha1.GitRepo, pollingInterval time.Duration) {
+				gitRepoCopy := gitRepo
+				gitRepoCopy.Spec.PollingInterval = &metav1.Duration{Duration: 10 * time.Second}
```

Comment:
If we're in a `same polling interval` case, does this line really belong here?

Reply:
It doesn't break the test because the `gitRepo` passed from `DescribeTable` has already the same duration, but, better to delete the line for clarity. 
Line deleted.